lib/email_notice.rb

```diff
@@ -0,0 +1,36 @@
+require "dotenv"
+require "pony"
+require_relative './database_connect'
+
+class EmailNotice
+
+  extend DatabaseConnect
+
+  def self.check_peep(peep, current_user_name)
+    result = database_connect.exec("SELECT user_name, email FROM users;")
+    result.each do |user|
```

Comment:
Rather than each, since you return on the first match, you might consider using 'any?' rather than 'each' - it makes it clearer what you're trying to achieve